config/cmd.yaml

```diff
@@ -84,4 +84,4 @@
       bytes: [0,15]
 
 # below is example of including additional command dictionary file
-- !include core_set_op_mode.yaml
+# - !include core_set_op_mode.yaml
```

Comment:
@jasonmlkang I know you filed a separate bug issue for this, but just wanted to understand it better.  Does this include work with the standard CmdDict use case?

Reply:
@nttoole It does not work with the default CmdDict specified in default config yaml.